patches/chrome-browser-android-chrome_feature_list.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/android/chrome_feature_list.cc b/chrome/browser/android/chrome_feature_list.cc
+index bfeea2227a320b1580868a64875d7497b8455fdb..cee091d8c6f409febd66a3f4ec42fa3cebf2b24e 100644
+--- a/chrome/browser/android/chrome_feature_list.cc
++++ b/chrome/browser/android/chrome_feature_list.cc
+@@ -231,6 +231,7 @@ const base::Feature* kFeaturesExposedToJava[] = {
+ };
+ 
+ const base::Feature* FindFeatureExposedToJava(const std::string& feature_name) {
++  BRAVE_FIND_FEATURE_EXPOSED_TO_JAVA
```

Comment:
https://github.com/brave/brave-core/commit/8ecdac337997f94a20227a541bc24515849a0cc3
This patch is added in 0.72.x and shouldn't be uplifted.

Reply:
That's why the android build failed as follows.
```
01:20:19  ../../chrome/browser/android/chrome_feature_list.cc:234:3: error: use of undeclared identifier 'BRAVE_FIND_FEATURE_EXPOSED_TO_JAVA'
01:20:19    BRAVE_FIND_FEATURE_EXPOSED_TO_JAVA
01:20:19    ^
01:20:19  1 error generated.
```

Reply:
oh no! Can you help me remove it? That would explain the Android failure 😅

Reply:
sure, on it.

Reply:
https://github.com/brave/brave-core/pull/3774/commits/e7a1361a81247e6b08d3b78a2fee88999fa6737f